.prettierrc.json

```diff
@@ -1,5 +1,9 @@
 {
+  "$schema": "http://json.schemastore.org/prettierrc",
+  "arrowParens": "avoid",
   "bracketSpacing": false,
+  "endOfLine": "lf",
```

Comment:
@giotramu why this?

Reply:
Sorry, I notice now that it's the default value. The goal was to force the Windows users to the line feed only.